server crashes after ~3 page reloads

(maybe only on my system, will try with updated node)

stack:

D:\BP Repos\lowfab\node_modules\browserify-middleware\lib\dynamic-cache.js:36
return [[filename]].concat(this._files[filename].map(getDeps)).reduce(fu
                                                       ^
RangeError: Maximum call stack size exceeded

npm ERR! Lowfab@0.4.0 start: `cake start`
npm ERR! Exit status 8
npm ERR!
npm ERR! Failed at the Lowfab@0.4.0 start script.
npm ERR! This is most likely a problem with the Lowfab package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cake start
npm ERR! You can get their info via:
npm ERR!     npm owner ls Lowfab
npm ERR! There is likely additional logging output above.
npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\Program Files\nodejs\\node.exe" "C:\Program Files\nod
ejs\node_modules\npm\bin\npm-cli.js" "start"
npm ERR! cwd D:\BP Repos\lowfab
npm ERR! node -v v0.10.35
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     D:\BP Repos\lowfab\npm-debug.log
npm ERR! not ok code 0

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "bootstrap": "^3.3.2",
     "bootstrap-styl": "^4.0.2",
     "browserify-data": "^0.2.0",
-    "browserify-middleware": "^5.0.1",
+    "browserify-middleware": "^4.1.0",
     "clone": "^1.0.0",
     "coffee-script": "^1.9.0",
     "coffeeify": "^1.0.0",

